docs/03-tutorials/00-application-connectivity/ra-01-establish-secure-connection-with-compass.md

```diff
@@ -15,7 +15,8 @@ To establish a secure connection with Compass and generate the client certificat
 
     To get the Connector URL and the one-time token which allow you to fetch the required configuration details, use the Compass Console.
 
-    >**NOTE:** To access the Compass Console, go to the `https://compass.{CLUSTER_DOMAIN}` URL and enter your Kyma credentials.
+   > [!NOTE]
+   > To access the Compass Console, go to the `https://compass.{CLUSTER_DOMAIN}` URL and enter your Kyma credentials.
```

Comment:
The three first notes (2 in step 1 and one in step 3) don't display well in GitHub.